docs/API.md

```diff
@@ -53,6 +53,21 @@ const subscription = barometer.subscribe(({ pressure }) =>
 
 Please note that this sensor has no option to set the update rate.
 
+## orientation: Observable<{qx: number, qy: number, qz: number, qw: number, pitch: number, roll: number, yaw: number, timestamp: string}>
+
+```js
+import { orientation } from "react-native-sensors";
+
+const subscription = orientation.subscribe(({ qx, qy, qz, qw, pitch, roll, yaw, timestamp }) =>
+  console.log({ qx, qy, qz, qw, pitch, roll, yaw, timestamp })
+);
+```
+
+qx, qy, qz, qw are quaternion values.
```

Comment:
Could you add a link to a place where one can read more about quaternion values and pitch, roll and yaw? (iOS / Android docs)

Reply:
Added